### ISSUE TITLE
About php files references

### ISSUE THREAD

**Issue by wazowski78** [yTakkar/Instagram-clone#2]
Hi again,

I am getting some troubles with the links of the pages. For example, if I try to load the index.php for the first time, my web explorer do not open it, the index file links to '/my_ig_root/welcome' and it does not loads, I have to enter manually '.php' extension to de URL to make it load.

I think I have missed something, and at this moment I can not figure it out, I have uploaded the project to my server, but I don't know if I have to use anything else.

Can you help me please?

So, the forms are not working for the same reason. Hope you can help!

**Comment by yTakkar** [comment]
I have used .htaccess file to configure the server so that extension is not allowed for any file.
Have a look in the .htaccess file.
secondly when I a user is not logged in, browser redirects to /rootfile/welcome page.
Hope that solves ur problem!!

**Comment by wazowski78** [comment]
Thank you very much for your reply. 

I can not find the htaccess file you are talking about. It is not added to the repository. 

I am having problems trying to make it work, I can follow the flow of the code, but I can't make it work, so, I think the problem is I do not know the starting point of the web app, I mean, system configuration and frameworks or needed components. 

I supone you are using your own server or VPS, and you have installed PHP, MySQL, etc... that's what I think I need to know to make it work for the first time, what do you think?

**Comment by yTakkar** [comment]
I am running it locally on localhost.
Yepp you're right I forgot to add .htaccess file I'll add it right now!!

**Comment by yTakkar** [comment]
One of the drawback of using PHP is there's no starting point like Isomorphic JS app that's why I've transitioned to JS completely.

**Comment by wazowski78** [comment]
Thank you again, I am migrating the project to a VPS, I hope I can get it work there. 

I really appreciate all your work, it is very impressive, may I ask you how did you code all this web? From scratch?

**Comment by yTakkar** [comment]
I followed all 3 instructions on readme.md and it worked like a charm!!

**Comment by wazowski78** [comment]
Hello again, I have follow some steps in order to make work in my server the .htaccess file, because by default it was disabled.
Now my problem is other, now I can load the web, but I only can see the 404.php page. So I think I have something wrong, with the htaccess file. Can you give me any tip for debug or fix it?

Thank you very much.

**Comment by yTakkar** [comment]
Well I didn't actually get your point, but it should have worked 'coz there are only 3 steps for it to work

**Comment by yTakkar** [comment]
problem is with `.htaccess` file only, because I've set a custom url whenever there's a 404 response.
So download `change_to_.htaccess.txt` file again and convert it into `.htaccess` and see if it works!!

**Comment by wazowski78** [comment]
No luck! I have tried to use as it is, and didn't work. I have edited, and the same thing, did not work either. I am migrating my sever to an another distribution, and I am going to try to set and test htaccess before upload the project. In the meantime, I have tried to remove the htaccess in all the files, but it seems that is not working properly. For example, registration does not work, and forgot password neither. I can not login because I can not register, I can edit the database to retrieve a new pass, but the forgot pass option does not work... I am a little stucked. I think, this is going to be my last chance to make it work. After this attempt I am take a break with it.
